integration/commands_diagnostic_test.go

```diff
@@ -299,6 +301,7 @@ func TestCommandsDiagnosticValidate(t *testing.T) {
 			"ok", float64(1),
 		))
 
+		actual.Remove("uuid")
```

Comment:
This looks like something we might want to add to our response too... (Of course, in a separate issue, but it is worth to add a TODO here)

Reply:
Comment with task #3841.

Reply:
Let's add a TODO comment above this line?

Reply:
Uh, that's odd... Because it's there!

Reply:
Maybe a GitHub caching glitch? If you go to the Files tab, you can see it.

<img width="802" alt="Screenshot 2023-12-13 at 09 03 12" src="https://github.com/FerretDB/FerretDB/assets/936421/c8aee008-df58-4d7e-8773-9fe9e8d4408e">

Reply:
ah, now I can see it!